.github/workflows/release.yaml

```diff
@@ -12,50 +12,194 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
 name: Release
 
-on:
-  release:
-    types:
-    - created
+run-name: Releasing ${{ inputs.target_version }} ${{ github.event.inputs.branch }}
+env:
+  REPO: quay.io
+  REPO_NAME: nuclio
+  CACHE_REPO: ghcr.io
+  CACHE_REPO_NAME: ${{ github.repository_owner }}
+  DOCKER_BUILDKIT: 1
 
-  # Run Release on push to development for unstable
+permissions:
+  contents: write
+  packages: write
+
+on:
   push:
     branches:
-    - development
-
+      - development
   workflow_dispatch:
     inputs:
+      version:
```

Comment:
rename this to `target_version` and remove the below `target_version`.
explicit is better IMO